agent/logger/field/constants.go

```diff
@@ -36,4 +36,5 @@ const (
 	Volume        = "volume"
 	Time          = "time"
 	NetworkMode   = "networkMode"
+	Tar           = "tar"
```

Comment:
I'm not sure if we will be referencing AppNet tar frequently enough log to merit adding it as a logger field, seems like only loader uses it... And also because the fields so far have been general task properties, but appnet tar name not so much. 
I think I'd probably rather just use formatted string in the few places we want to log the tar file name.